src/frontend/components/style.scss

```diff
@@ -50,3 +53,68 @@ $select:#272727;
         }
     }
 }
+.formlogin ul>li{
```

Comment:
Lets separate the login styles into login.scss file. so create a new file inside public and paste your styling there